src/main/webapp/resources/js/types/irida/index.d.ts

```diff
@@ -2,71 +2,153 @@ export = IRIDA;
+    collectionDate: Date;
+    geographicLocationName: string;
+    isolationSource: string;
+    latitude: string;
+    longitude: string;
+    projects: Project[];
+    sequenceFiles: any[]; // TODO (Josh - 6/7/22): FLush this out
+    defaultSequencingObject: number | null;
+    defaultGenomeAssembly: number | null;
+  }
 
-    export type SystemRole =
-        "ROLE_ANONYMOUS"
-        | "ROLE_ADMIN"
-        | "ROLE_USER"
-        | "ROLE_MANAGER"
-        | "ROLE_SEQUENCER"
-        | "ROLE_TECHNICIAN";
+  export type SystemRole =
+    | "ROLE_ANONYMOUS"
+    | "ROLE_ADMIN"
+    | "ROLE_USER"
+    | "ROLE_MANAGER"
+    | "ROLE_SEQUENCER"
+    | "ROLE_TECHNICIAN";
 
-    interface User extends IridaBase {
-        username: string;
-        email: string;
-        firstName: string;
-        lastName: string;
-        phoneNumber: string;
-        enabled: boolean;
-        systemRole: SystemRole;
-        lastLogin: Date;
-        locale: string;
-        projects: Project[];
-        tokens: string[]; // TODO (Josh - 6/7/22): Look into this one
-        announcements: Announcement[];
-        subscriptions: string[]; // TODO (Josh - 6/7/22): Look into this one as well
-    }
-}
\ No newline at end of file
+  interface User extends IridaBase {
+    username: string;
+    email: string;
+    firstName: string;
+    lastName: string;
+    phoneNumber: string;
+    enabled: boolean;
+    systemRole: SystemRole;
+    lastLogin: Date;
+    locale: string;
+    projects: Project[];
+    tokens: string[]; // TODO (Josh - 6/7/22): Look into this one
+    announcements: Announcement[];
+    subscriptions: string[]; // TODO (Josh - 6/7/22): Look into this one as well
+  }
+
+  export type ProcessingState =
+    | "UNPROCESSED"
+    | "QUEUED"
+    | "PROCESSING"
+    | "FINISHED"
+    | "ERROR";
+
+  interface SequencingFile {
+    createdDate: Date;
+    file: string;
+    fileName: string;
+    identifier: string;
+    label: string;
+    links: [];
+    modifiedDate: Date;
+    uploadSha: string;
+  }
+
+  interface SequencingObject {
+    createdDate: Date;
+    fileProcessor: string;
+    files: SequencingFile[];
+    forwardSequenceFile: SequencingFile;
+    identifier: string;
+    label: string;
+    links: [];
+    processingState: ProcessingState;
+    reverseSequenceFile: SequencingFile;
+    sequenceFile: SequencingFile;
+  }
+
+  interface SampleSequencingObject {
+    fileInfo: SequencingObject;
+    secondFileSize: string;
+    firstFileSize: string;
+    processingState: ProcessingState;
+    fileType: string;
+    file: any | undefined; //TODO: (deep - 08/08/22): Flush this out
+    qcEntries: any;
+  }
+
+  interface SampleGenomeAssembly {
+    fileInfo: {
+      createdDate: Date;
+      file: string;
+      fileName: string;
+      fileRevisionNumber: number;
+      identifier: string;
+      label: string;
+      links: [];
+    };
+    fileType: string;
+    firstFileSize: string;
+  }
+
+  export type AnalysisStateType =
```

Comment:
I was not add `Type` to the end of definitions in here, they are to match to the server.  And I was trying to keep this file alphabetized

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f